src/main/java/core/basesyntax/Zoo.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Zoo {
+
+    private AviaryForBirds aviaryForBirds;
+    private Aviary aviary;
+    private Aquarium aquarium;
+
+    public Zoo() {
+
+        Shark shark = new Shark();
+        Sparrow sparrow = new Sparrow();
+        Penguin pengiun = new Penguin();
+        Tiger tiger = new Tiger();
+
+        this.aviaryForBirds = new AviaryForBirds(sparrow);
+        this.aviary = new Aviary(tiger, pengiun);
+        this.aquarium = new Aquarium(shark);
+    }
+
+    public void giveEatforEnimals() {
+
+        List list = new ArrayList();
```

Comment:
it's a bad practice to use List with raw parameters 
```suggestion
        List<Animal> list = new ArrayList<>();
```